DataConnectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_api.py

```diff
@@ -67,4 +67,14 @@ def get_triage_comments(self, item_id):
 
         items_url = self.url + "triage-item-comments?id=" + str(item_id)
         response = requests.get(items_url, headers={"Authorization": "Basic %s" % self.b64val, "searchlight-account-id": "%s" % self.id})
-        return response.text
\ No newline at end of file
+        return response.text
+    
+    def get_triage_events_by_num(self, event):
+        """
+            gets triage events by number
+        """
+        event = event - 1
+        triage_url = self.url + "triage-item-events?limit=20&event-num-after=" + str(event)
+        response = requests.get(triage_url, headers={"Authorization": "Basic %s" % self.b64val, "searchlight-account-id": "%s" % self.id})
```

Comment:
Can we add User agent in the header of all DS calls. Check with Poornima, she has recently done it.